src/core/lifetime.d

```diff
@@ -931,51 +945,6 @@ version (CoreUnittest)
     }
 }
 
-/+ these tests can't be performed in druntime, but a mirror still exists in phobos...
```

Comment:
These tests are still in `std.conv`. I don't think this comment and the 2 disabled tests provide any value whatsoever here in druntime and see it rather as confusing clutter.